cluster-scope/base/user.openshift.io/groups/curator/group.yaml

```diff
@@ -8,3 +8,4 @@ users:
   - matankb
   - leihchen
   - ajinkyababar
+  - avrajath
```

Comment:
@AvRajath Adding github handle here will be sufficient for curator access in smaug